ncm-spma/src/main/perl/spma/ips.pm

```diff
@@ -364,7 +392,7 @@ sub run_pkg_command
     #   (4 for /usr/bin/pkg, 1 for /usr/bin/spma-run)
     #
     my ($self, $cmd, $log, $exit_void) = @_;
-    my $proc = CAF::Process->new($cmd, log => $self);
+    my $proc = CAF::Process->new($cmd, log => $self, keeps_state => 1);
```

Comment:
at the very least make it clear in the comment that it is not to be used for just any command, ideally renaming the function, like `query_pkg_command` or somethign that suggests `keeps_state`. no need for new args.

wrt the otehr remarks, esp if you are using this already, consider them informational, and up to you to change any code. given you have no unittests, modifying code might do more dmage than good (esp considering MS is the only user of ips).